core/src/executor/evaluate/function.rs

```diff
@@ -501,6 +502,45 @@ pub fn prepend<'a>(expr: Evaluated<'_>, value: Evaluated<'_>) -> Result<Evaluate
     }
 }
 
+pub fn sort<'a>(expr: Evaluated<'_>, order: Evaluated<'_>) -> Result<Evaluated<'a>> {
+    let expr: Value = expr.try_into()?;
+    let order: Value = order.try_into()?;
+
+    match expr {
+        Value::List(mut l) => match order {
```

Comment:
how about adapting the similar strategy which we do in `sort` module?
in the `sort.rs` execution module, (https://github.com/gluesql/gluesql/blob/main/core/src/executor/sort.rs)

`Value` does not impl `Ord` but `Key` does, so for `Value::List(Vec<Value>)`,
1. convert the list value to contain `Key` - `Vec<(Key, Value)>`
2. then sort it.
3. remove `Key`

fyi. the ideal way to resolve this would be.. to use `Value::evaluate_cmp` but currently the `evaluate_cmp` impl in `Value` returns `Option<Ordering>` so it is not proper to be used for sorting.
for now, the realistic option would be to use `Key`. but I will consider to update `Value::evaluate_cmp` to return `Ordering`

Reply:
https://github.com/gluesql/gluesql/pull/1300/commits/0f860a04c4e149ec3ec7429d0e92026380d93907

I understood that adding each `Value` of the list as a `Key`, and sorting using `Key::cmp`
Is it right what you intended?

Reply:
Is there anything else I need to fix?